tests/test_validation.py

```diff
@@ -711,7 +711,7 @@ def validate_hypotest(pdf, data, mu_test, expected_result, tolerance=1e-6):
     [
         (setup_1bin_shapesys(), 1e-6),
         (setup_1bin_lumi(), 4e-6),
-        (setup_1bin_normsys(), 1e-6),
+        (setup_1bin_normsys(), 2e-6),
```

Comment:
increased the tolerance for now - no idea why the discrepancy increased here. Also in this case the discrepancy does not become significantly lower when using the minuit optimizer ...

Reply:
i think i'd be fine with this increase (or alternatively we re-base it to the new numebers as a reference)

Reply:
I would be wary about just rebasing to the new numbers. What if we added another test with the new numbers and tighter tolerances?

Reply:
Everything else looks great. This is the last missing piece.

Reply:
I'm running some tests right now, but I agree with @kratsg. This is a beautiful PR, so once we double check this is needed we're GTG!

Reply:
As we're talking about at `1e-6` change, is it possible that the difference in representation is enough in some evaluation cases to bump this? I think this is fine though.